pkg/informers/informers.go

```diff
@@ -92,7 +92,9 @@ func NewInformersForResourceWithTransform(ifs FactoriesForNamespaces, resource s
 func PartialObjectMetadataStrip(obj interface{}) (interface{}, error) {
 	partialMeta, ok := obj.(*v1.PartialObjectMetadata)
 	if !ok {
-		return nil, fmt.Errorf("internal error: cannot cast object %#+v to PartialObjectMetadata", obj)
+		// Don't do anything if the cast isn't successful.
```

Comment:
nit: we can add a link to this if you want: https://github.com/kubernetes/kubernetes/blob/5909bd2ddf9b4a4ee2c6507be7dc50d1920e5035/staging/src/k8s.io/client-go/tools/cache/delta_fifo.go#L141-L160

Reply:
Good remark! I've updated the function comment.